decidim-core/app/views/decidim/participatory_processes/_process.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="column">
+  <article class="card card--process">
+    <div class="card__image-top"
+      style="background-image:url(<%= participatory_process.banner_image.url %>)"></div>
+    <div class="card__content">
+      <%= link_to participatory_process_path(participatory_process), class: "card__link" do %>
+        <h4 class="card__title"><%= translated_attribute participatory_process.title %></h4>
+      <% end %>
+      <p class="card__desc"><%= translated_attribute(participatory_process.short_description).html_safe %></p>
```

Comment:
Same as said in https://github.com/codegram/decidim/pull/108/files#diff-9b704086d7cac076073ce4d1a4b52c71R8